shared/teams/team/invites/invite-history.tsx

```diff
@@ -0,0 +1,192 @@
+        ),
+        title: <ModalTitle title="Invite links" teamID={teamID} />,
+      }}
+      footer={{
+        content: Styles.isMobile ? (
+          <Kb.Button fullWidth={true} mode="Secondary" label="Generate link" onClick={onGenerate} />
+        ) : (
+          <Kb.Button fullWidth={true} type="Dim" label="Close" onClick={onClose} />
+        ),
+        hideBorder: Styles.isMobile,
+      }}
+      onClose={onClose}
+      allowOverflow={true}
+      mode="DefaultFullHeight"
+      noScrollView={true}
+    >
+      <Kb.Tabs
+        tabs={[{title: activeTitle}, {title: expiredTitle}]}
+        onSelect={title => setShowingExpired(title === expiredTitle)}
+        selectedTab={showingExpired ? expiredTitle : activeTitle}
+        style={styles.tabs}
+      />
+      {emptyOrLoading ?? (
+        <Kb.BoxGrow>
+          <Kb.SectionList
+            sections={sections}
+            keyExtractor={item => item.id}
+            renderItem={({item}) => <InviteItem inviteLink={item} teamID={teamID} />}
+            contentContainerStyle={styles.listContent}
+            stickySectionHeadersEnabled={true}
+          />
+        </Kb.BoxGrow>
+      )}
+    </Kb.Modal>
+  )
+}
+
+const InviteItem = React.memo(
+  ({inviteLink, teamID}: {inviteLink: Types.InviteLink; teamID: Types.TeamID}) => {
+    const dispatch = Container.useDispatch()
+    const yourUsername = Container.useSelector(s => s.config.username)
+    const onExpire = () => dispatch(TeamsGen.createRemovePendingInvite({inviteID: inviteLink.id, teamID}))
+
+    const duration = formatDurationLong(new Date(), new Date(inviteLink.expirationTime * 1000))
+    // TODO Y2K-1715 - when expired we should show how long the invite link was valid for.
+    const expireText = inviteLink.expired ? `Expired ${duration} ago` : `Expires in ${duration}`
+
+    return (
+      <Kb.Box2 direction="vertical" style={styles.inviteContainer} gap="xtiny">
+        <Kb.CopyText text={inviteLink.url} disabled={inviteLink.expired} />
+        <Kb.Box2 direction="vertical" fullWidth={true}>
+          <Kb.Text type="BodySmall">
+            Invites as {inviteLink.role} • {expireText}
+          </Kb.Text>
+          <Kb.Text type="BodySmall">
+            Created by{' '}
+            {inviteLink.creatorUsername === yourUsername ? (
+              'you'
+            ) : (
+              <Kb.ConnectedUsernames
+                inline={true}
+                colorFollowing={true}
+                type="BodySmall"
+                usernames={inviteLink.creatorUsername}
+              />
+            )}{' '}
+            • {inviteLink.numUses.toLocaleString()} joined
+            {!!inviteLink.lastJoinedUsername && (
+              <Kb.Text type="BodySmall">
+                , most recently{' '}
+                <Kb.ConnectedUsernames
+                  inline={true}
+                  colorFollowing={true}
+                  type="BodySmall"
```

Comment:
Cècile says all colorFollowing usernames should be bold. Perhaps we should make TS enforce that.